### PR TITLE
Add notes on Bower

### PR DESCRIPTION
**Pull request by elizabrock**

### CODE REVIEW

File: front-end-notes/bower.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "version": "0.0.1", 
+  "dependencies": { 
+    "sass-bootstrap": "~3.0.0", 
+    "modernizr": "~2.6.2", 
+    "jquery": "~1.10.2" 
+  },
+  "private": true 
+}
+```
+
```
Comment (elizabrock) [inline front-end-notes/bower.md]: There's extra whitespace here.  :(

**Comment by tombler** [comment]
Might want to include a note about using `sudo` with npm install -g bower. Other than that looks dope!

**Comment by sqfish** [comment]
You may also want to add a note about using bower to update packages.

File: front-end-notes/bower.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Installing Bower
+ Bower is a command line utility. Install it with npm.<br />
+  <br />
+ `$ npm install -g bower`<br />
+ <br />
+*Bower requires node, npm and git.*
+## Installing Packages
+**Install packages individually** <br />
+<br />
+`$ bower intall <package name>`<br />
```
Comment (sqfish) [inline front-end-notes/bower.md]: There is a typo here: install is misspelled

Comment (wfhutch) [inline front-end-notes/bower.md]: Typo here on "intall"

File: front-end-notes/bower.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+# Bower 
+Bower is a package manager for the web.
+> Web sites are made of lots of things — frameworks, libraries, assets, utilities, and rainbows.
```
Comment (NewEvolution) [inline front-end-notes/bower.md]: I almost went and googled rainbows. :laughing:

File: front-end-notes/bower.md (hunk 2 above)
Comment (NewEvolution) [inline front-end-notes/bower.md]: Mention the _--save_ option to save the installed package into the _bower.json_ file

**Pull request merged**